soql-standalone-parser/src/test/scala/com/socrata/soql/parsing/ParserTest.scala

```diff
@@ -231,29 +231,43 @@ class ParserTest extends WordSpec with MustMatchers {
 
     "window function over partition order round trip" in {
       val x = parseFull("select row_number() over(partition by x, y order by m, n)")
-      x.selection.expressions.head.expression.toString must be ("row_number() OVER ( PARTITION BY `x`,`y` ORDER BY `m` NULL LAST, `n` NULL LAST)")
+      x.selection.expressions.head.expression.toString must be ("row_number() OVER ( PARTITION BY `x`, `y` ORDER BY `m` ASC NULL LAST, `n` ASC NULL LAST)")
```

Comment:
so I get that ASC is the default if not specified, but I am a little curious why these examples started specifying it?

Reply:
previously, we don't support asc/desc.  Full ordering support is new.